.editorconfig

```diff
@@ -0,0 +1,15 @@
+# editorconfig for VS code
```

Comment:
`editorconfig` is obvious
`VS` is unnecessary restrictive -- editorconfig is used by lots of different tools, including non-Microsoft tools AFAICT
`code` is misleading (this is a configuration file for the VS editor/IDE; the code is ours)

```suggestion
# https://editorconfig.org/
```
 or

```suggestion
```